app/views/visual_groups/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_with(model: [@visual_model, @visual_group], local: true) do |form| %>
```

Comment:
I suggest giving the form an html `id` to help with javascript, and also with integration tests. I'm starting to do that everywhere in my refactoring:

```
form_with(model: [@visual_model, @visual_group], local: true,
          id: "visual_group_form")
```